✨ Add agenda items for special biweekly/monthly scrum meetings (#39)
phillymedia/data-engineering-handbook: handbook/sprint-process.md

Author: @Rosi-D I forget which of these standups were going to be slightly longer, but we can note that here

- Expectations:
  - Punctual participation with transparent and open communication

##### Biweekly/Monthly Scrum Agenda Items

To ensure team alignment and transparency, certain scrum meetings will have additional agenda items, as follows:

> - **Monday scrum before sprint launch (biweekly)**

Reviewer: proposed text change: Monday scrum meeting before sprint launch (biweekly)

>   - Team members evaluate tickets that we anticipate spilling over into the next sprint. In-progress tickets are cloned and re-pointed to reflect estimated spillover work. Tickets with status of To Do or Blocked are rescheduled, as appropriate.

Reviewer: Proposed text change: Team members evaluate tickets anticipated to spill over into the next sprint. In-progress tickets are cloned and re-pointed to reflect the estimated remaining work.

>   - Team members may edit the point value of their assigned tickets to better reflect their actual level-of-effort.

Reviewer: Proposed text change: Team members may adjust the point values of their assigned tickets to better reflect their actual level-of-effort.

> - **Tuesday standup before sprint launch (biweekly)**

Reviewer: Proposed Text Change:  Tuesday scrum meeting before sprint launch (biweekly)

>   - Team review of the upcoming Sprint Planning Log, with a focus on:

Reviewer: Add: This scrum meeting will be extended by 15 minutes to accommodate the following:
Team review of the upcoming Sprint Planning Log .....

    - General epics and work themes
    - Risks
    - OOO and handoff information
    - Special ceremonies (e.g. Dev Day, D&I Demo)

- **Thursday standup, 1 week before D&I Demo day (monthly)**
  - Team members commit to a presentation topic for the following week's D&I Demo session. They may draw from any of their work since the previous D&I Demo.

> - **Monday standup, week of Dev Day (monthly)**

Reviewer: proposed text change:  Monday scrum meeting, week of Dev Day (monthly)

- Expectations:
  - Punctual participation with transparent and open communication

##### Biweekly/Monthly Scrum Agenda Items

To ensure team alignment and transparency, certain scrum meetings will have additional agenda items, as follows:

- **Monday scrum meeting before sprint launch (biweekly)**
  > [!NOTE]  
  > This scrum meeting will be extended by 15 minutes.

>   - Team members evaluate tickets that we anticipate spilling over into the next sprint. In-progress tickets are cloned and re-pointed to reflect estimated spillover work. Tickets with status of To Do or Blocked are rescheduled, as appropriate.

Author: ```suggestion
  - Team ahhahahahahahahaha. In-progress tickets are cloned and re-pointed to reflect estimated spillover work. Tickets with status of To Do or Blocked are rescheduled, as appropriate.
```
